Gems/ScriptAutomation/Code/Source/ScriptAutomationSystemComponent.cpp

```diff
@@ -395,4 +401,58 @@ namespace AZ::ScriptAutomation
             });
     }
 
+    void ScriptAutomationSystemComponent::LoadLevel(const char* levelName)
+    {
+        AZ_Printf("ScriptAutomation" "Level load requested of %s\n", levelName);
+        AZ_Assert(!m_levelLoading, "Attempting to load a level while still waiting for a level to load");
+        m_levelLoading = true;
+        m_levelName = levelName;
+        AzFramework::LevelSystemLifecycleNotificationBus::Handler::BusConnect();
+        PauseAutomation();
+
+        auto loadLevelString = AZStd::string::format("LoadLevel %s", levelName);
+
+        AzFramework::ConsoleRequestBus::Broadcast(
+            &AzFramework::ConsoleRequests::ExecuteConsoleCommand, loadLevelString.c_str());
+
+    }
+
+    void ScriptAutomationSystemComponent::OnLevelNotFound(const char* levelName)
+    {
+        AZ_Printf("ScriptAutomation" "OnLevelNotFound of %s\n", levelName);
+        if (m_levelName == levelName)
+        {
+            m_levelLoading = false;
+            m_levelName = "";
+            AZ_Error("ScriptAutomation", false, "Level not found \"%s\"", levelName);
+            AzFramework::LevelSystemLifecycleNotificationBus::Handler::BusDisconnect();
+            DeactivateScripts();
+        }
+    }
+
+    void ScriptAutomationSystemComponent::OnLoadingComplete(const char* levelName)
+    {
+        AZ_Printf("ScriptAutomation" "OnLoadingComplete of %s\n", levelName);
+        if (m_levelName == levelName)
+        {
+            m_levelLoading = false;
+            AzFramework::LevelSystemLifecycleNotificationBus::Handler::BusDisconnect();
+            ResumeAutomation();
+        }
+    }
+
+    void ScriptAutomationSystemComponent::OnLoadingError(const char* levelName, [[maybe_unused]] const char* error)
+    {
+        AZ_Printf("ScriptAutomation" "OnLoadingError of %s\n", levelName);
```

Comment:
These printfs should be removed or changed to an `AZ_Info` message as it is only for tracing purposes.

Reply:
I've removed the debug printf's